src/frontend-admin/src/components/reusables/Card/Card.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styles from './Card.module.css'
+import {Title} from '../Title/Title' 
+
+interface ICardProps {
+    title: any
+    children: any //any so that dropdown can be used
+}
+
+export const Card: React.FC<ICardProps> = (props) => {
+    const { title, children } = props
+
+    return (
+        <div className={styles.cardMain}>
+            <Title title={title}/>            
```

Comment:
If our title prop is optional, you can change this line to be `{title && <Title title={title} />}` Conditional rendering like that is probably my favorite aspect of jsx :)